src/InfoBubble/InfoBubble.js

```diff
@@ -0,0 +1,86 @@
+import { ClickAwayListener } from '@mui/material';
+import Tooltip from '@mui/material/Tooltip';
+import { dcnb } from 'cnbuilder';
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import { Button } from '../Button/Button';
+import { alignOptions } from './InfoBubble.levers';
+
+export const InfoBubble = ({
+  alignment,
+  className,
+  children,
+  text,
+  ...props
+}) => {
+  const [open, setOpen] = useState(false);
+
+  const handleTooltipClose = () => {
+    setOpen(false);
+  };
+
+  const handleTooltipOpen = () => {
+    setOpen(true);
+  };
+
+  const bubbleStyles = {
+    tooltip: dcnb(
+      className,
+      '!su-px-18 !su-pb-18 !su-pt-12 !su-sans !su-text-14 !su-text-black !su-bg-black-10 !su-color-black-10 !su-shadow !su-max-w-[25ch]'
```

Comment:
Our Decanter TW classes seem to be overridden by the MUI styles without the TW important prefix. 🤔